ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -1314,6 +1314,10 @@ def verify_consumer_storagecluster(sc_data):
         "\\d+(\\.\\d+){3}:31659",
         sc_data["spec"]["externalStorage"]["storageProviderEndpoint"],
     )
+    log.info(
+        f"Onboarding ticket begins with: {sc_data['spec']['externalStorage']['onboardingTicket'][:50]}"
+    )
+    assert len(sc_data["spec"]["externalStorage"]["onboardingTicket"]) > 500
```

Comment:
If the assert fails then content of `sc_data["spec"]["externalStorage"]["onboardingTicket"]` is logged into test results. Could it be a problem?

Reply:
I think it's not a problem, the logs would contain only small part of the ticket (10% if the token is 500 chars long, so one would have to still guess remaining 450 chars which is quite a lot, moreover the token would be actually even longer).

That said, I would rather log only 10 chars instead of 50, as it would make a nicer log line.

Reply:
@mbukatov fixed

Reply:
I would also add the last ten chars of the token to have more information.

Reply:
@yitzhak12 ok, done

Reply:
Okay, thanks.